crates/subspace-archiving/src/archiver.rs

```diff
@@ -840,6 +840,10 @@ impl Archiver {
             object_mapping,
         }
     }
+
+    pub fn kzg(&self) -> Kzg {
```

Comment:
Why? This is an internal property and shouldn't be exposed.

Reply:
To quickly obtain `Kzg` to reinitilize the archiver. I'm open to a better way to get this data.

Reply:
If you just need Kzg instance you can acquire it the same exact way as it is done during original archiver initialization. It is quite ugly to just expose random internals because you need them elsewhere. There should be a clean (as much as possible) dependency injection tree, etc.

Reply:
I'm not sure what you wanted to say with this comment. Kzg instance is the same everywhere. Yes, it does have some cache for efficiency reasons, but it is not unique or special in any way. We just clone `Kzg` and pass it along. There is no need to expose it from archiver itself.